ApiBundle/Tests/Functional/Controller/AreaControllerTest.php

```diff
@@ -21,6 +21,9 @@ public function testAreaReverseTransform()
         $json = json_decode($this->client->getResponse()->getContent(), true);
         $area = $json['areas'][1];
         $this->assertSame('myMain', $area['area_id']);
+        if (!array_key_exists(0, $area['areas'])) {
+            $this->markTestIncomplete('Datas have been altered');
+        }
```

Comment:
More than marking the test as incomplete wouldn't it be more logical to stop the tests and display a message to the user saying that he must reload the fixtures ?
I mean if you want to run the tests, you have to run ALL the tests. That second pass is meaningless if some tests aren't runned.